### DISCUSSION TITLE
Add Clone, Add, and Save to the fullscreen resource editor

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#3767] [Feature Requests]
## How it looks now:
<img width="961" alt="image" src="https://github.com/specify/specify7/assets/37256050/1f901b31-75e3-4180-8e07-e8831b00ca43">

## How it should look:
<img width="956" alt="image" src="https://github.com/specify/specify7/assets/37256050/c75518d9-18e7-4af4-bbc8-8cff26ee1c56">

It's inconvenient to need to leave full screen, click save, then go back into full screen editing. The "Close" button is redundant since there is a fullscreen icon in the top level menu. It's logical to click the button you clicked to go full screen to minimize again.

**Comment by maxpatiiuk** [comment]
I think this also fixes a UX problem with the current design: having just a `Close` button, and no `Save` button makes users feel like their changes won't be saved.

Also, the `Close` button was redundant with the full screen exit button.

I think we should definitely do the change you proposed, assuming it is not too difficult in the code (I don't remember the code, but probably the 3 buttons could just be assigned to a variable, and that variable could be put in a different place depending if you are in full screen or not)
